src/Font Dialog/main.vala

```diff
@@ -0,0 +1,37 @@
+#! /usr/bin/env -S vala workbench.vala --pkg gtk4
+
+private Gtk.FontDialog dialog_custom;
+
+async void on_clicked () {
+    try {
+        var family = yield dialog_custom.choose_family (workbench.window, null, null);
+
+        message (@"Font Family: $(family.get_name())");
+    } catch (Error err) {
+        warning ("Error: %s\n", err.message);
```

Comment:
Use string formatting, please. Additionally, `message`, `warning` and `critical` already add a line break after the message, so you don't need to add one your self.
```suggestion
        warning (@"Error: $(err.message)");
```